src/components/AddItems.js

```diff
@@ -90,67 +91,76 @@ const AddItemsToList = () => {
 
   return (
     <div>
-      <div>
-        <form onSubmit={submitShoppingListItemHandler}>
-          <h2>Add Item to List</h2>
-          {shoppingListItemNameExists ? (
-            <p>
-              {`You have ${normalizeString(
-                shoppingListItemName,
-              )} in your shopping list already`}
-            </p>
-          ) : null}
-          <div>
-            <label>
-              Name
-              <input
-                type="text"
-                value={shoppingListItemName}
-                onChange={shoppingListItemNameHandler}
-              />
-            </label>
-          </div>
-          <fieldset>
-            How soon are you likely to buy it again?
+      <div className="max-h-screen flex flex-col box-border items-center">
+        <header className="bg-green-400 w-full fixed text-center">
```

Comment:
In the description you say that you picked bg-green-400 for visibility reasons. Did the other green color contrast badly?

Reply:
Yes, it was too light, the heading would be difficult to read. But this information was especially for the other team since we first decided to go with green-300 (I thought I would mention it in the PR, but as I see this makes it confusing for others who do not know what we decided in the first place).